src/js/plugins/plugin.chapters.js

```diff
@@ -21,14 +21,29 @@ BookReader.prototype.setup = (function (super_) {
   };
 })(BookReader.prototype.setup);
 
-
 /** @override Extend to call Open Library for TOC */
 BookReader.prototype.init = (function(super_) {
   return function() {
     super_.call(this);
     if (this.enableChaptersPlugin && this.ui !== 'embed') {
       this.getOpenLibraryRecord();
     }
+    if (this.enableMobileNav) {
+      this.bind(BookReader.eventNames.mobileNavOpen,
+        () => {
+          this.updateTOCState(this.firstIndex, this._tocEntries);
+
+          if($('#mm-4').hasClass('mm-opened')){
```

Comment:
Is there another selector we could use instead of `mm-4`? That will change if there are other panels added in the future.

Reply:
done